cmd/serve/httplib/serve/dir.go

```diff
@@ -77,26 +79,36 @@ func (d *Directory) Serve(w http.ResponseWriter, r *http.Request) {
 	defer accounting.Stats.DoneTransferring(d.DirRemote, true)
 
 	fs.Infof(d.DirRemote, "%s: Serving directory", r.RemoteAddr)
+
+	var indexPage = GetTemplate(w)
```

Comment:
I would say we don't want to be instantiating the template every time this function is called.

I would instantiate it in `httplib.NewServer` and store the indexTemplate in the `Server` struct.